src/pages/cli/graphql-transformer/config-params.mdx

```diff
@@ -116,7 +116,7 @@ Follow these two steps when you need to rotate an API Key
 
 ## ElasticSearchInstanceCount
```

Comment:
@danielleadams Are any of these config name params changing?

Reply:
No.